docs/dev/developer-guides/aa.md

```diff
@@ -1,11 +1,8 @@
 # Account abstraction support
 
 ::: warning
-
-Please note that with system update released in Feb 2023, the ergs concept is only used by the VM and the SDK (version `0.13.0` and above) whilest the API layer operates with gas. 
-
-Find [more information in the changelog](../troubleshooting/changelog.md).
-
+- Please note that with the system update released in Feb 2023, the `ergs` concept is only used by the VM while the API layer operates with `gas`. 
+- For more information, read the [changelog](../troubleshooting/changelog.md).
```

Comment:
we should link to the #system-update-v1-3-feb-8th-2023 part to be more precise and friendly

Reply:
also, this warning box with this message appears in many other places... worth to update them all

Reply:
We have a ticket for this: https://linear.app/matterlabs/issue/DOC-184/remove-ergs-warnings-across-the-docs-apart-from-changelog

Reply:
DONE